--- origsrc/s2tc-f6ec862d7594e29ae80a6e49f66ad4c76cf09abc/Makefile.am	2015-12-27 16:40:35.000000000 -0600
+++ src/s2tc-f6ec862d7594e29ae80a6e49f66ad4c76cf09abc/Makefile.am	2016-05-05 13:04:27.698966000 -0500
@@ -27,9 +27,10 @@ endif
 if ENABLE_LIB
 lib_LTLIBRARIES = libtxc_dxtn.la
 libtxc_dxtn_la_SOURCES = s2tc_algorithm.cpp s2tc_libtxc_dxtn.cpp s2tc_common.h s2tc_algorithm.h txc_dxtn.h s2tc_license.h
-libtxc_dxtn_la_LDFLAGS = -avoid-version -nodefaultlibs
+libtxc_dxtn_la_LDFLAGS = -avoid-version -no-undefined
 libtxc_dxtn_la_LIBADD = -lm
 libtxc_dxtn_la_CFLAGS = -fvisibility=hidden -Wold-style-definition -Wstrict-prototypes -Wsign-compare -Wdeclaration-after-statement
+libtxc_dxtn_la_CXXFLAGS = -fno-exceptions
 library_includedir = $(includedir)
 library_include_HEADERS = txc_dxtn.h
 pkgconfigdir = $(libdir)/pkgconfig
